### ISSUE TITLE
Beepsky can stun/arrest while switched off.

### ISSUE THREAD

**Issue by Cerebulon** [PolarisSS13/Polaris#3318]
#### Brief description of the issue
Officer Beepsky is able to stun and handcuff people adjacent to it and who attack it while its status is set to "Off". 

#### What you expected to happen
Whacking beepsky would destroy it without it fighting back.

#### What actually happened
After a few hits, beepsky played his 'arrest' voice line and stunbatoned me, then handcuffed me on the ground.

#### Steps to reproduce
Switch beepsky "Off".
Attack with object several times.

#### Additional info:
master - 2017-04-16
d09cc7a57dc3aaa809707f386541d42ccbb246d1

**Comment by Anewbe** [comment]
I think this may have been an admin thing going on.

**Comment by Cerebulon** [comment]
That's possible. When I tried to recreate this, I couldn't get beepsky to attack me at all unless I set myself to arrest in records, but I don't know if that was something to do with my command ID.
I also tried to fix it (And didn't bother in the end, since I couldn't recreate it) and there is a secondary much more minor issue that it still runs the 'react to attack' code when he's switched off as far as I could tell. Which seems to include playing the voice line, and reporting the attack to... Security?
